### ISSUE TITLE
brew install --cask git-credential-manager-core fails

### ISSUE THREAD

**Issue by kyrichen55** [GitCredentialManager/git-credential-manager#585]
After running 
```
brew tap microsoft/git
brew install --cask git-credential-manager-core
```
I receive the following error:
```
==> Running installer for git-credential-manager-core; your password may be nece
Package installers may write to any location; options such as `--appdir` are ignored.
installer: Package name is Git Credential Manager Core
installer: Installing at base path /
installer: The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “gcmcore-osx-2.0.605.12951.pkg”.)
==> Purging files for version 2.0.605.12951 of Cask git-credential-manager-core
Error: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/env LOGNAME=myUser USER=myUser USERNAME=myUser /usr/sbin/installer -pkg /usr/local/Caskroom/git-credential-manager-core/2.0.605.12951/gcmcore-osx-2.0.605.12951.pkg -target / -allowUntrusted` exited with 1. Here's the output:
installer: Package name is Git Credential Manager Core
installer: Installing at base path /
installer: The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “gcmcore-osx-2.0.605.12951.pkg”.)
``` 

I have also attempted to install the package through the .pkg file on the releases page, but I also receive an error that prevents it from installing.

**Comment by c4milo** [comment]
I'm running into this as well.

**Comment by spoilsportmotors** [comment]
The last package that actually installs for me is 2.0.498; I kept going backwards from current, and that was the first that worked. I haven't looked to see what changed in the installation scripts, but it appears to be something towards the tail end. I'm on macOS 11.6.2.

**Comment by mjcheetham** [comment]
Are you running on a new M1 Mac or an Intel Mac? There is a known issue at the moment installing/upgrading GCM on M1 Macs via Homebrew: https://github.com/GitCredentialManager/git-credential-manager/issues/574

**Comment by FarisCS** [comment]
Running into this on 
MacBook pro 2015
Big Sur 11.6

**Comment by ldennington** [comment]
@FarisCS - glad you're unblocked. When you say upgrading, do you mean upgrading your OS?

**Comment by FarisCS** [comment]
Yes the OS. But I suspect it had something to do with xcode since running brew doctor gave me a warning about xcode version being (unknown) and outdated

**Comment by ldennington** [comment]
Got it. I'm going to close this issue given updating XCode (and potentially the OS as well) mitigates it.

**Comment by jankytay** [comment]
Maybe it's because the package wasn't notarized?
<img width="723" alt="Screen Shot 2022-04-22 at 4 55 06 PM" src="https://user-images.githubusercontent.com/5750485/164669687-6d948e43-25e0-4901-adad-bb2dfa6c4b38.png">

**Comment by jankytay** [comment]
nvm the installer returned the same error after manually approved from Settings. But I did found the error in console
<img width="754" alt="Screen Shot 2022-04-22 at 5 10 35 PM" src="https://user-images.githubusercontent.com/5750485/164675411-6a801ca5-fe6e-461d-beb0-3bef246cc5c2.png">

**Comment by lsp85** [comment]
> Running into this on MacBook pro 2015 Big Sur 11.6
> 
> Update:
> 
> After upgrading and _updating xcode_ the gcm was installed with no hiccups.

That did for me as well!

Using a:

- Mac Mini 2020
- M1 
- macOS Ventura 13.5.1
